Examples/Swift/UserTrackingModesExample.swift

```diff
@@ -115,25 +119,40 @@ class UserLocationButton : UIButton {
         return bezierPath.cgPath
     }
     
-    func updateArrow(for mode: MGLUserTrackingMode) {
-        var stroke: CGColor
-        switch (mode) {
+    // Update the arrow's color and rotation when
+    // tracking mode is changed
```

Comment:
plzzzzzzzzz make this one line and add a period.

Reply:
🤔 I have this locally but even after committing and pushing it didn't appear. I'll try again.